src/charm.py

```diff
@@ -47,7 +48,11 @@ def _on_install(self, _):
         subprocess.check_output(["service", "nginx", "stop"])
         os.remove("/etc/nginx/nginx.conf")
         os.unlink("/etc/nginx/sites-enabled/default")
-        create_path()
+
+        if not os.path.exists(SSL_PATH):
+            os.makedirs(SSL_PATH, 0o755)
```

Comment:
```suggestion
        os.makedirs(SSL_PATH, 0o755, exist_ok=True)
```

Adding `exist_ok=True` allows us to remove a conditional. :)